.github/workflows/release.yml

```diff
@@ -0,0 +1,214 @@
+        id: release
+        uses: googleapis/release-please-action@v4
+        with:
+          token: ${{ steps.get-release-bot-token.outputs.token }}
+
+    outputs:
+      prs_created: ${{ steps.release.outputs.prs_created }}
+      releases_created: ${{ steps.release.outputs.releases_created }}
+      sha: ${{ steps.release.outputs.sha }}
+      tag_name: ${{ steps.release.outputs.tag_name }}
+
+  merge:
+    if: needs.release.outputs.prs_created == 'true' && needs.release.outputs.releases_created == 'false'
+    name: Merge Release PR
+    needs: release
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      contents: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: release-please--branches--main
+
+      - name: Get Release Bot Token
+        id: get-release-bot-token
+        uses: peter-murray/workflow-application-token-action@v1
+        with:
+          application_id: ${{ secrets.RELEASE_BOT_ID }}
+          application_private_key: ${{ secrets.RELEASE_BOT_PRIVATE_KEY }}
+          organization: "Lilypad-Tech"
+
+      # Approve with github-actions bot
+      - name: Approve PR
+        id: approve
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          gh pr review --approve
+
+      # Auto-merge PR when status checks complete or merge immediately if no checks in progress
+      # The auto-merge flag requires branch protection rule overrides.
+      - name: Merge Release PR
+        env:
+          GH_TOKEN: ${{ steps.get-release-bot-token.outputs.token }}
+        run: gh pr merge --squash --auto || gh pr merge --squash
+
+  extend-notes:
+    if: needs.release.outputs.releases_created == 'true' && needs.release.outputs.prs_created == 'false'
+    name: Extend Release Notes
+    needs: release
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      contents: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ needs.release.outputs.tag_name }}
+
+      - name: Get Release Bot Token
+        id: get-release-bot-token
+        uses: peter-murray/workflow-application-token-action@v1
+        with:
+          application_id: ${{ secrets.RELEASE_BOT_ID }}
+          application_private_key: ${{ secrets.RELEASE_BOT_PRIVATE_KEY }}
+          organization: "Lilypad-Tech"
+
+      - name: Extend Release Notes
+        env:
+          GH_TOKEN: ${{ steps.get-release-bot-token.outputs.token }}
+        run: |
+          BODY=$(gh release view ${{ needs.release.outputs.tag_name }} --json body | jq -r '.body')
+          EXTENSION=$(cat .github/releases/release_notes.md)
+
+          # Build up notes from BODY and EXTENSION seperated by two newlines
+          # https://trstringer.com/github-actions-multiline-strings/#option-2---environment-variable
+          NOTES=$(cat << EOF
+          $BODY
+
+          $EXTENSION
+          EOF
+          )
```

Comment:
NOTES=$(cat << EOF
          $BODY
          $EXTENSION
          EOF
)

Reply:
Yeah, admittedly this is an esoteric technique for adding the newlines. But it was the only one that worked out for me implementing it. 😄